README.md

```diff
@@ -313,6 +313,14 @@ To add Prometheus exposition to an existing HTTP server, see the `MetricsHandler
 which provides a `BaseHTTPRequestHandler`. It also serves as a simple example of how
 to write a custom endpoint.
 
+To enable TLS identity verification, `certfile` and `keyfile` need to be provided.
+
```

Comment:
In addition, I think we should describe how the use of mTLS is controlled when enabling HTTPS.

How about:

```
When HTTPS is enabled, if `cafile` is provided, mutual TLS (mTLS) is used and enforced,
i.e. the prometheus client requires that Prometheus presents a client certificate during
the TLS handshake, which the prometheus client validates using the specified CA certificates.
```

Reply:
More detailed documentation would really help others understand and use this feature. Thank you for your review.